config/base.json

```diff
@@ -37,7 +37,6 @@
 	},
 	"flags": {
 		"enable_datadog": true,
-		"enable_io_beta_cta": true,
-		"enable_hcp_vault_secrets_api_docs_revision": false
```

Comment:
The `enable_hcp_vault_secrets_api_docs_revision` flag is no longer used, so I've removed it completely 👍

Reply:
Thanks for cleaning this up